components/Sidebar/sidebar.module.css

```diff
@@ -0,0 +1,155 @@
+.wrapper {
+  font: 700 16px 'DM Sans', sans-serif;
+  margin-bottom: 25px;
+  width: 20vw;
+  height: max-content;
+  margin-right: auto;
+}
+
+.sidebar {
+  height: calc(100vh - 78px);
+  overflow-y: scroll;
+  background-color: #f8f7ff;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  flex-direction: column;
+}
+
+.sidebar::-webkit-scrollbar {
+  width: 0.5px;
+}
+.sidebar::-webkit-scrollbar-thumb {
+  visibility: hidden;
+}
+
+.sidebar > * {
+  padding: 1.6rem;
+  width: 100%;
+}
+
+.heading {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 10px;
+}
+
+.options{
+  gap: 1rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.option {
+  position: relative;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 6px 9px;
+  gap: 10px;
+  color: #a5a2b8;
+  cursor: pointer;
+}
+
+.option:hover {
+  color: #352e5bd0;
+}
+
+.option_image {
+  position: static;
+  color: #352e5b;
+}
+
+.option_active {
+  color: #352e5b;
+}
+
+.option_bar {
+  width: 3px;
+  aspect-ratio: 1/6;
+  background-color: #9381ff;
+  border-radius: 1px 0px 0px 1px;
```

Comment:
You can use shorthand property for border radius: 
Refer here: https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius